src/OrchardCore/OrchardCore.DisplayManagement.Liquid/HtmlContentValue.cs

```diff
@@ -91,15 +91,4 @@ public override void WriteTo(TextWriter writer, TextEncoder encoder, CultureInfo
     {
         _value.WriteTo(writer, (HtmlEncoder)encoder);
     }
-
-    [Obsolete("GetValue is obsolete, prefer the GetValueAsync method.")]
```

Comment:
Breaking changes, right?

Reply:
We don't. The original code here was a mistake.